modules/covid_vaccine/app/services/covid_vaccine/v0/facility_lookup_service.rb

```diff
@@ -57,6 +57,7 @@ def nearest_facilities(lat, lng)
         }
       rescue
         # For now just bail on any exception while getting facilities
+        # TODO Add Sentry logging
```

Comment:
Just checking if you want to add Sentry logging or if it's ok to move forward without the logging for the time being?